<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serialdialog</class>
 <widget class="QDialog" name="serialdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#serialdialog
{
background-image: url(:/t507_button_image/all/background.jpg);
}
QDialog {border:1px solid gray;}
QListWidget {background-color:transparent; outline: none; border:1px solid gray; color: black; }</string>
  </property>
  <widget class="QComboBox" name="serialModeBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>client</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>server</string>
    </property>
   </item>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>421</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>serial port</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>serial set</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="serialOkBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="serialCancelBtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>330</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>mode choose</string>
   </property>
  </widget>
  <widget class="QPushButton" name="serialCheckAllBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>check all</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
